datakit-ci.opam

```diff
@@ -9,12 +9,12 @@ dev-repo:     "https://github.com/docker/datakit.git"
 doc:          "https://docker.github.io/datakit/"
 
 build: [
-  "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "false" "-n" name
+  "ocaml" "pkg/pkg.ml" "build" "--dev-pkg" "%{pinned}%" "--tests" "false" "-n" name
```

Comment:
Not needed, but https://github.com/dbuenzli/topkg/blob/master/CHANGES.md#v090-la-forclaz-vs says `Deprecate --pinned in favor of --dev-pkg in the build command`.